core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -55,6 +55,9 @@ public abstract class LoggingDecoratorBuilder {
         };
     }
 
+    @Nullable
+    private final Logger defaultLogger;
```

Comment:
Question) Couldn't we just set to `logger` instead of introducing a new field?

Reply:
Yes, we can. 👍